<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddNode</class>
 <widget class="QDialog" name="DialogAddNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>50</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Parent</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parentList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>50</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tagList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>72</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddNode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Node</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>50</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textNode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
